extensions/amp-story/1.0/amp-story-quiz.css

```diff
@@ -0,0 +1,145 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins&display=swap');
+
+:host {
+    align-self: start;
+    --correct-color: #5BBA74 !important;
+    --correct-color-shaded: #2D8E42 !important;
+    --incorrect-color: #F34E4E !important;
+    --incorrect-color-shaded: #cc4141 !important;
+    --accent-color: #005AF0;
+  }
+
+.i-amp-story-quiz-container {
+    font-family: 'Poppins-bold', sans-serif !important;
+    background: var(--accent-color) !important;
+    border-radius: 32px !important;
+}
+
+.i-amp-story-quiz-prompt-container {
+    display: grid !important;
+    justify-items: center !important;
+    padding: 20px !important;
+    color: white !important;
+}
+
+.i-amp-story-quiz-prompt {
+    margin: 0px !important;
+}
+
+h1.i-amp-story-quiz-prompt {
+    font-size: 28px !important;
+    line-height: 36px !important;
+}
+
+h2.i-amp-story-quiz-prompt {
+    font-size: 22px !important;
+    line-height: 28px !important;
+}
+
+h3.i-amp-story-quiz-prompt {
+    font-size: 18px !important;
+    line-height: 24px !important;
+}
+
+.i-amp-story-quiz-option-container {
+    display: grid !important;
+    background-color: white !important;
+    border-radius: 32px !important;
+    padding: 8px !important;
+    padding-bottom: 0px !important;
+}
+
+.i-amp-story-quiz-option {
+    display: inline-grid !important;
+    grid-template-columns: 50px auto !important;
+    justify-items: start !important;
+    align-items: center !important;
+    border-radius: 32px !important;
+    padding: 8px !important;
+    padding-right: 16px !important;
+    background-color: inherit !important;
+    margin-bottom: 8px !important;
+    color: #5F6368 !important;
+    border: solid 1px !important;
+    border-color: #DADCE0 !important;
+    /* TODO: Why won't this notation work? */
+    /* font: 16px/20px !important; */
+    font-size: 16px !important;
+    line-height: 20px !important;
+}
```

Comment:
What happens if the user enters 10 lines of text?
Please add a max-height and some ellipsis in case of text overflow: https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow

Reply:
In progress, I added some CSS to support this case but I still have an issue where the ellipsis doesn't trigger until repaint - working on fixing that now

Reply:
Wow I fixed it by adding `visibility: visible` to the CSS as someone did in [this StackOverflow article](https://stackoverflow.com/questions/38989475/css-multi-line-line-clamp-ellipsis-doesnt-work). Big thanks to Eloy Ruiz here.